sdx_controller/messaging/rpc_queue_consumer.py

```diff
@@ -3,25 +3,33 @@
 import logging
 import os
 import threading
+import time
 from queue import Queue
 
 import pika
 
 from sdx_controller.handlers.lc_message_handler import LcMessageHandler
 from sdx_controller.utils.parse_helper import ParseHelper
 
-MQ_HOST = os.environ.get("MQ_HOST")
 # subscribe to the corresponding queue
 SUB_QUEUE = os.environ.get("SUB_QUEUE")
-
+MQ_SRVC = os.environ.get("MQ_SRVC")
+MQ_USER = os.environ.get("MQ_USER")
+MQ_PASS = os.environ.get("MQ_PASS")
 logger = logging.getLogger(__name__)
 
 
 class RpcConsumer(object):
     def __init__(self, thread_queue, exchange_name, topology_manager):
         self.logger = logging.getLogger(__name__)
+        SLEEP_TIME = 5
```

Comment:
Why do we a need to add a delay anyway?

I suspect this is to wait for the RabbitMQ service to start up, in the case of RabbitMQ and SDX Controller spawned by the same compose file.  If that is the case, we should be using [depends_on](https://docs.docker.com/compose/compose-file/05-services/#depends_on) in the compose file.

Reply:
Hi @sajith, docker depends on is only responsible for managing the status of Docker services, not the internal services running inside the container, such as RabbitMQ. To avoid repetition, I plan to add a new variable to the .env file, as we will use this value multiple times in sdx lc and controller.

Reply:
I do not understand. The message queue would not be running in the same container.  What is the delay for?